README.md

```diff
@@ -157,7 +157,13 @@ for documentation on building mutli-platform images with Docker. You can change
 # PLATFORMS=linux/arm64,linux/amd64 make docker-buildx
 ```
 
-2. Deploy the controller and daemonset with the default images. All required CRDs will be installed by this command:
+2. Deploy the cert manager
+
+```console
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.14.3/cert-manager.yaml
```

Comment:
It seems a mandatory manual step. Do we want to make it part of the setup script (i.e. a pre-requisite step for deploying the controller)?

Reply:
Any particular reason to install v1.14.3? The latest release is v1.15.3.
Also, maybe we should consider installing with Helm 
> cert-manager provides Helm charts as a first-class method of installation on both Kubernetes and OpenShift

Reply:
@empovit Good question, I am not sure if open shift deployments will consume this cert-manager. talking about the deploy script,  the script is generally used for KinD deployments, hence I agree it can be part of a script, please test it with the latest version of cert-manager and send a PR. I hope you agree that this task would be orthogonal to getting this PR merged.

Reply:
Currently the README addresses only Kind deployments, so we should either stick to that, or document a more generic procedure (and move away from the opinionated script). WDYT?

Reply:
ok I vote for README to be specific for KinD, many users who have this repo do not have access to the OpenShift cluster.